### DISCUSSION TITLE
Update schedule?

### DISCUSSION THREAD

**Discussion by MysticDaedra** [lllyasviel/stable-diffusion-webui-forge#581] [General]
The last commits were made 2-ish weeks ago. 2 weeks is a long time in AI space, even longer for a UI that purports to be a bleeding-edge optimization fork. Is there a stated update schedule somewhere, or a link to a dev branch or something that is getting updates more frequently?

**Comment by jepjoo** [comment]
Mr. Zhang has many projects going and is definitely working on something currently but dunno if related to Forge.

I'm hoping Stability has given him access to SD3 and he is busy working on that. :)

**Reply by 9wow** [reply]
Looking through his activity, lots of work in 'private repository'. so maybe 👀

**Comment by ghost** [comment]
Hm... I have to be honest, I also wonder if there is an update schedule and what the future dev plan for Forge looks like.

It's been ~3 weeks since the last commit and many features are still not complete, don't work or are completely missing. I've spent dozens of hours working with Forge in the last few weeks and there are many bugs and crashes too if you start doing advanced stuff with it.

Yes, I know the project is free and don't get me wrong, I am very grateful to the author and all the people involved. But I agree with you OP - you can't ignore the fact that 3 weeks is a long time in the AI field, especially for a project thats kind of a bleeding edge fork compared to regular A1111.

For Forge (or A1111 like UIs in general) to be able to remain competitive and a good alternative to ComfyUI and other UIs, it desperately needs more and frequent updates. I fear it will die otherwise.

For example, I ended up switching back to normal A1111 and am currently getting familiar with ComfyUI, although I don't even really like it. But I have little choice if I want a lightweight, resource efficient and powerful UI that gets all the new shit and bug fixes in a timely manner.

Forge has so much potential but is (at least for me) not really usable at the moment because so many features are missing or broken.

**Reply by rotatorotator** [reply]
Same to me, I am going to swich back to the A1111 dev branch because of the stagnaion in development here. A1111 has caught up with forge, so in my case I do not see any need to stay on forge webui.

**Reply by strawberrymelonpanda** [reply]
In terms of quality of life features rather than memory / arch features, A1111 was never actually behind.  Forge is just based off the pre-release Dev branch of Auto, rather than the Main branch.

The main reason I'm still with Forge as an 8GB VRAM user is memory and speed. As well as a few new tricks like [LayerDiffuse](https://github.com/layerdiffusion/sd-forge-layerdiffuse).  Since Forge is based off Dev though, I really would like to see the last 3 weeks of A1111 [development](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commits/dev/) pulled in.

**Comment by slann303** [comment]
Will also switch back to auto1111, Controlnet IPAdapter doesn't work at the moment with Adetailer here, no updates or comments regarding bugs.

**Comment by E2GO** [comment]
@lllyasviel Is it possible to get some info on next updates, please?

**Comment by strawberrymelonpanda** [comment]
Honestly I'd settle for routine pulls from the [A1111 Dev](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commits/dev/) branch into Forge, which is currently 4 weeks behind.  

The "Development Plan" thread states:
> We will get all updates from the dev branch of original webui automatically with bots,

That doesn't seem to be happening.  I'd also understand if the project switched from Dev to Main releases, but that should probably be communicated.

**Comment by xhoxye** [comment]
The problem is that every project of the author stops updating

**Reply by E2GO** [reply]
It's not one author as I remember - there's a whole team. And sometimes they goes for another big projects and vacation. It was a couple of times already. So it just could be it.

**Comment by Vinzero-SS** [comment]
So can we get any confirmation? Is this project just dead?

It's not really worth staying with Forge in the long run for extra speed if it's going to be this far behind, especially once new types of model formats come out that Forge just won't have the compatibility for.

**Reply by E2GO** [reply]
As I remember they said that updates for Forge will be deployed only for most critical bugs and updates. So may be there's nothing worth of it yet?... Don't know - just thoughts...

**Reply by strawberrymelonpanda** [reply]
When / where was this?

**Reply by E2GO** [reply]
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/166#discussion-6202513

**Comment by VantomPayne** [comment]
And A1111 just updated to 1.9.0

**Reply by thiagojramos** [reply]
After the debate about whether Forge was based on ComfyUI or not, Forge died.

**Comment by reosablo** [comment]
See #801
